site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3753,7 +3753,7 @@ public function getSubmittersWhoGotMarkBySection($mark, $grader, $gradeable, $an
         // Switch the column based on gradeable team-ness
         $type = $mark->getComponent()->getGradeable()->isTeamAssignment() ? 'team' : 'user';
         // TODO: anon teams?
-        $user_type = ($type == 'user' && $anon != 'unblind') ? 'anon' : $type;
+        $user_type = ($type == 'user' && $anon !== 'unblind') ? 'anon' : $type;
```

Comment:
```suggestion
        $user_type = ($type === 'user' && $anon !== 'unblind') ? 'anon' : $type;
```

Should use more strict comparison